h2o-automl/src/test/java/hex/ModelStubs.java

```diff
@@ -0,0 +1,43 @@
+package hex;
+
+import ai.h2o.automl.targetencoding.TargetEncoderModel;
+import org.junit.Ignore;
+import water.H2O;
+import water.Key;
+
+import java.io.IOException;
+
+@Ignore
+public interface ModelStubs {
+  
+  public class TestModel extends TargetEncoderModel {
```

Comment:
the name of the class is confusing

Reply:
@honzasterba agree, TargetEncoderTestModel will be better